velox/common/memory/RawVector.h

```diff
@@ -127,12 +139,14 @@ class raw_vector {
 
   void reserve(int32_t size) {
     if (capacity_ < size) {
-      T* newData = allocateData(size, capacity_);
+      T* newData = allocateData(size);
+      const auto newCapacity = calculateCapacity(size);
       if (data_) {
         memcpy(newData, data_, size_ * sizeof(T));
-        freeData(data_);
+        free();
       }
       data_ = newData;
+      capacity_ = newCapacity;
```

Comment:
allocateData already update capacity_?

Reply:
Yeah we need to remove the update in allocateData.